lnwallet/channel.go

```diff
@@ -1567,6 +1567,10 @@ func (lc *LightningChannel) remoteLogUpdateToPayDesc(logUpdate *channeldb.LogUpd
 	case *lnwire.UpdateFulfillHTLC:
 		ogHTLC := localUpdateLog.lookupHtlc(wireMsg.ID)
 
+		if ogHTLC.addCommitHeightLocal == 0 {
+			ogHTLC.addCommitHeightLocal = commitHeight - 1
```

Comment:
Comment should also explain why we can use commitheight-1 here, even though that not might be the actual add hight.

Reply:
I should determine if it's necessary to set `commitHeight - 1` rather than `commitHeight`. Pretty sure it doesn't matter.